apps/nextra/pages/en/build/smart-contracts/prover/spec-lang.mdx

```diff
@@ -857,6 +837,14 @@ module 0x42::m {
 }
 ```
 
+Notice that type parameters are supported in global invariants. For example:
```

Comment:
nit: perhaps we could add a brief high-level description in English about what the invariant is specifying here, may be as a comment above the invariant or here in prose.

Reply:
Updated the doc to use the example above